components/Cards/src/ShapeThemer/MDCCardsShapeThemer.m

```diff
@@ -27,8 +27,10 @@ + (void)applyShapeScheme:(id<MDCShapeScheming>)shapeScheme
 
 + (id<MDCShapeGenerating>)cardShapeGeneratorFromScheme:(id<MDCShapeScheming>)shapeScheme {
   MDCRectangleShapeGenerator *rectangleShape = [[MDCRectangleShapeGenerator alloc] init];
-  MDCCornerTreatment *cornerTreatment = shapeScheme.mediumSurfaceShape.topLeftCorner;
-  [rectangleShape setCorners:cornerTreatment];
+  rectangleShape.topLeftCorner = shapeScheme.mediumSurfaceShape.topLeftCorner;
```

Comment:
Do we know that `topRightCorner`, `topLeftCorner`, `bottomLeftCorner`, `bottomRightCorner` will have a value? If a user sets `topRightCorner`, `bottomLeftCorner` but does nothing to `topLeftCorner` or `bottomRightCorner` they will still have a value right?

Reply:
yes, that's fine. not setting a value is also acceptable and expected. No value means they haven't customized that corner in any way.